src/main.ml

```diff
@@ -100,9 +100,8 @@ let open_in_editor refresh log_lines = function
             match Sys.getenv_opt "EDITOR" with
             | Some x -> [ x ]
             | None -> (
-                match Sys.getenv_opt "PAGER" with
-                | Some x -> [ x ]
-                | None -> [] ) )
+                match Sys.getenv_opt "PAGER" with Some x -> [ x ] | None -> [] )
```

Comment:
Surprising: a closing parenthesis alone on a line.

Also the match can fit on a single line but it would be at least as readable on a multiple ones.